scripts/cxg_admin.py

```diff
@@ -63,6 +65,68 @@ def delete_dataset(ctx, uuid):
             exit(0)
 
 
+@cli.command()
+@click.argument("collection_uuid")
+@click.argument("new_owner")
+@click.pass_context
+def update_collection_owner(ctx, collection_uuid, new_owner):
+    """Update the owner of a corpora collection. You must first SSH into the target deployment using
+    `make db/tunnel` before running."""
+
+    with db_session_manager() as session:
+
+        key = (collection_uuid, CollectionVisibility.PUBLIC.name)
+        collection = Collection.get(session, key)
+        collection_name = collection.to_dict()["name"]
+
+        if collection is not None:
+            click.confirm(
+                f"Are you sure you want to update the owner of the collection:{collection_name}?",
+                abort=True,
+            )
+            collection.update(owner=new_owner)
+            collection = Collection.get(session, key)
+            if collection and (collection.owner == new_owner):
+                click.echo(f"Updated owner of collection:{collection_uuid}. Owner is now {collection.owner}")
+                exit(0)
+            else:
+                click.echo(f"Failed to update owner for collection_uuid:{collection_uuid}")
+                exit(0)
+
+
+@cli.command()
+@click.argument("curr_owner")
+@click.argument("new_owner")
+@click.pass_context
+def transfer_collections(ctx, curr_owner, new_owner):
+    """Transfer all collections owned bu the curr_owner to the new_owner. You must first SSH into the target
+    deployment using `make db/tunnel` before running."""
+
+    with db_session_manager() as session:
+        collections = session.query(DbCollection).filter(DbCollection.owner == curr_owner).all()
+
+        if collections is not None:
+            click.confirm(
+                f"Are you sure you want to update the owner of {len(collections)} collection from {curr_owner} to "
```

Comment:
```suggestion
                f"Are you sure you want to update the owner of {len(collections)} collections from {curr_owner} to "
```

Reply:
if you wanna be extra fancy 
```suggestion
                f"Are you sure you want to update the owner of {len(collections)} collection{'s' if len(collections)>1 else ''} from {curr_owner} to "
```